### PR TITLE
Fix revision question

### PR DESCRIPTION
**Pull request by sourabhv**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python/core/sequential-data-types-ii/list-methods-ii.md [hunk 1]
```diff
@@ -70,19 +70,22 @@ keyboard.???()
 
 
 ---
-
 ## Revision
```
Comment (sourabhv) [inline python/core/sequential-data-types-ii/list-methods-ii.md:73]:
```suggestion

## Revision
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/core/sequential-data-types-ii/list-methods-ii.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/python/core/sequential-data-types-ii/list-methods-ii.md?ref=9bfcf3b675dd1fb95836a141dc6cdf81e4e5fcec)

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**